css-frontend/src/main/java/org/sonar/css/tree/symbol/SymbolVisitor.java

```diff
@@ -90,6 +89,7 @@ private void enterScope(Tree tree) {
   @Override
   public void visitLessVariable(LessVariableTree tree) {
     Usage.Kind usage;
+    System.out.println("eee");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)